README.md

```diff
@@ -282,3 +284,13 @@ expected to honor this code.
 
   [code-of-conduct]: https://github.com/spotify/code-of-conduct/blob/master/code-of-conduct.md
   [maven-search]: https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.spotify%22%20hamcrest*
+
+## Ownership
+
+The Weaver squad is currently owning this project internally.
```

Comment:
Do we need more explanation of what this team is for outsiders? This is an internal Spotify team name.

Reply:
@davidxia we thought about it just because we wanted to set up expectations internally about the ownership, we also put it at the bottom of the README so it's not that intrusive for people outside Spotify